demo/src/screens/componentScreens/ButtonsScreen.tsx

```diff
@@ -234,15 +234,16 @@ export default class ButtonsScreen extends Component {
               style={{marginBottom: ButtonSpace}}
               iconSource={iconStyle => (
                 <View
-                  style={{
-                    width: 20,
-                    height: 20,
-                    // @ts-expect-error
-                    backgroundColor: iconStyle[0]?.tintColor,
-                    borderRadius: 10,
-                    // @ts-expect-error
-                    marginRight: iconStyle[0]?.marginRight
-                  }}
+                  style={[
```

Comment:
Let's add all the usages we found for passing tintColor (in iconStyle, in iconProps) so we'll have senety check. What do you think?